shell/src/main/java/alluxio/shell/command/TailCommand.java

```diff
@@ -85,11 +98,32 @@ void runCommand(AlluxioURI path, CommandLine cl) throws IOException {
 
   @Override
   public String getUsage() {
-    return "tail <path>";
+    return "tail -C <number of bytes> <path>";
   }
 
   @Override
   public String getDescription() {
-    return "Prints the file's last 1KB of contents to the console.";
+    return "Prints the file's last user specified number of bytes (by default, 1KB) of contents "
+        + "to the console.";
+  }
+
+  @Override
+  protected Options getOptions() {
+    Option bytesOption =
+        Option.builder("C")
+              .required(false)
+              .numberOfArgs(1)
+              .desc("user specified option")
+              .build();
+    return new Options().addOption(bytesOption);
+  }
+
+  @Override
+  public boolean validateArgs(String... args) {
```

Comment:
Thanks for the review. @aaudiber 
If we want this command line to be something like, alluxio fs tail -c 1024 ${path}, I think that we need override this function. Otherwise, it will not recognize this, and throw out exception.

But if we want the command line to be something like, alluxio fs tail ${path} ${numOfBytes}, we do not need to override it.